### PR TITLE
plugins/sqlite-lua: init

### PR DESCRIPTION
**Pull request by khaneliman**
Creating for https://github.com/kkharji/sqlite.lua to support work being done in https://github.com/nix-community/nixvim/pull/2114#discussion_r1759770838

### CODE REVIEW

File: plugins/by-name/sqlite-lua/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "sqlite-lua";
+  originalName = "sqlite.lua";
+  luaName = "sqlite.lua";
+  package = lib.mkPackageOption pkgs "sqlite-lua" {
+    default = [
+      "vimPlugins"
+      "sqlite-lua"
+    ];
+  };
```
Comment (MattSturgeon) [inline plugins/by-name/sqlite-lua/default.nix:15]:
```suggestion
  package = "sqlite-lua";
```

**Review by MattSturgeon — COMMENTED** [review]
Everything seems good

File: plugins/by-name/sqlite-lua/default.nix (hunk 1 above)
Comment (khaneliman) [reply plugins/by-name/sqlite-lua/default.nix:15]: Oh yeah, the main package option is a little smarter, isn't it?

**Review by MattSturgeon — APPROVED** [review]

**Comment by khaneliman** [comment]
@mergifyio queue

**Pull request merged**